workflow/setup_expt.py

```diff
@@ -73,7 +73,7 @@ def fill_COMROT_cycled(host, inputs):
     rdatestr = datetime_to_YMDH(inputs.idate - to_timedelta('T06H'))
     idatestr = datetime_to_YMDH(inputs.idate)
 
-    if os.path.isdir(os.path.join(inputs.icsdir, 'model_data', 'atmos')):
+    if os.path.isdir(os.path.join(inputs.icsdir, f'{inputs.cdump}.{rdatestr[:8]}', rdatestr[8:], 'model_data', 'atmos')):
```

Comment:
I think this was supposed to be `idatestr` not `rdatestr`. If you create cold start ICs from ufs_utils, there won't be a rdate directory.